roles/couchbase-infra/meta/main.yml

```diff
@@ -0,0 +1,60 @@
+galaxy_info:
```

Comment:
This boilerplate can be removed... I am sure I need to remove this from mine too....

Reply:
Will remove.